pkg/collector/corechecks/cluster/ksm/kubernetes_state.go

```diff
@@ -745,17 +744,24 @@ func ownerTags(kind, name string) []string {
 	return tags
 }
 
-// podLabelsMapperOverride defines the label mapper overrides
-// that should be applied on pod metrics only.
-var podLabelsMapperOverride = map[string]string{"phase": "pod_phase"}
```

Comment:
This is a bit out of scope but multiple check instances can run concurrently by the same agent - global variables shouldn't be used. see https://github.com/DataDog/datadog-agent/pull/11652